pkgs/development/libraries/audio/rtmidi/default.nix

```diff
@@ -1,14 +1,14 @@
 { stdenv, fetchFromGitHub, autoconf, automake, libtool, libjack2, alsaLib, pkgconfig }:
 
 stdenv.mkDerivation rec {
-  version = "2.1.1";
+  version = "3.0.0";
```

Comment:
From [RtMidi page](http://www.music.mcgill.ca/~gary/rtmidi/):

> The version number has been bumped to 3.0.0 because of the past API change concerning the renaming of the RtError class to RtMidiError.

That might be problematic, I'll check if this bump breaks other packages and if it does, I'll add a separate derivation for v3...

Reply:
I don't know how to do that by now. Not a long term NixOS user. But I create a ticket to learn that too.

Reply:
This can be done via `nix-shell -p nox --run "nox-review 33132"`.

Reply:
I tried that, but I get "merge conflicts" (dont know why, and where) I'm using nixos stable (17.09 I think). So I can't do these checks I guess